gymnasium/experimental/wrappers/__init__.py

```diff
@@ -86,3 +86,68 @@
     "VectorRecordEpisodeStatistics",
     "VectorListInfo",
 ]
+
+
+class DeprecatedWrapper(ImportError):
+    """Exception raised when an old version of a wrapper is imported."""
+
+    pass
+
+
+class InvalidVersionWrapper(ImportError):
+    """Exception raised when an invalid version of a wrapper is imported."""
+
+    pass
+
+
+def __getattr__(wrapper_name):
+    """Raise errors when importing deprecated or invalid versions of wrappers.
+
+    Args:
+        wrapper_name (str): The name of the wrapper being imported.
+
+    Raises:
+        DeprecatedWrapper: If the version is not the latest.
+        InvalidVersionWrapper: If the version doesn't exist.
+        ImportError: If the wrapper does not exist.
+    """
+    if wrapper_name == "__wrapped__" or "__test__":
+        raise AttributeError(
+            f"module 'gymnasium.experimental.wrappers' has no attribute {wrapper_name}"
+        )
+
+    base_name = wrapper_name[:-1]
+    version = wrapper_name[-1]
+
+    # Get all wrappers that start with the base wrapper name
+    wrappers = [name for name in globals().keys() if name.startswith(base_name)]
+
+    # If the wrapper does not exist, raise an ImportError
+    if not wrappers:
+        raise ImportError(
+            f"cannot import name '{wrapper_name}' from 'gymnasium.experimental.wrappers'"
+        )
+
+    # Get the latest version of the wrapper
+    latest_version = max([int(name[-1]) for name in wrappers])
```

Comment:
I don't hope for this to happen but we should probably support version numbers over 10

Reply:
I see, but it would be handier in the future to add a symbol between the name and the version of the wrapper, like with the environments. So the version checking would be more straighfoward

Reply:
I understand that but we decided that we should only use camel case and not mix camel and snake case, i.e., `NormalizeReward_v0`